src/main/kotlin/hs/kr/equus/user/domain/auth/service/PassPopupService.kt

```diff
@@ -0,0 +1,106 @@
+package hs.kr.equus.user.domain.auth.service
+
+import hs.kr.equus.user.domain.auth.presentation.dto.request.PassPopupRequest
+import hs.kr.equus.user.global.exception.InternalServerErrorException
+import hs.kr.equus.user.global.utils.pass.RedirectUrlChecker
+import kcb.module.v3.OkCert
+import org.json.JSONObject
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class PassPopupService(
+    private val redirectUrlChecker: RedirectUrlChecker
+) {
+
+    companion object {
+        private const val TARGET = "PROD"
+    }
+
+    @Value("\${pass.site-name}")
+    private lateinit var siteName: String
+
+    @Value("\${pass.site-url}")
+    private lateinit var siteUrl: String
+
+    @Value("\${pass.popup-url}")
+    private lateinit var popupUrl: String
+
+    @Value("\${pass.cp-cd}")
+    private lateinit var cpCd: String
+
+    @Value("\${pass.license}")
+    private lateinit var license: String
+
+    private val svcName = "IDS_HS_POPUP_START"
+
+    private val rqstCausCd = "00"
+
+    @Transactional
+    fun execute(passPopupRequest: PassPopupRequest): String {
+        redirectUrlChecker.checkRedirectUrl(passPopupRequest.redirectUrl)
+        try {
+            val reqJson = JSONObject()
+            reqJson.put("RETURN_URL", passPopupRequest.redirectUrl)
+            reqJson.put("SITE_NAME", siteName)
+            reqJson.put("SITE_URL", siteUrl)
+            reqJson.put("RQST_CAUS_CD", rqstCausCd)
+
+            val reqStr: String = reqJson.toString()
+
+            val okcert = OkCert()
+
+            val resultStr: String = okcert.callOkCert(TARGET, cpCd, svcName, license, reqStr)
+
+            val resJson = JSONObject(resultStr)
+
+            val RSLT_CD: String = resJson.getString("RSLT_CD")
```

Comment:
🚫 **[ktlint]** <standard:property-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Property name should start with a lowercase letter and use camel case